pkg/config/config.go

```diff
@@ -228,6 +228,7 @@ func InitConfig(config Config) {
 	config.BindEnvAndSetDefault("python_version", DefaultPython)
 	config.BindEnvAndSetDefault("allow_arbitrary_tags", false)
 	config.BindEnvAndSetDefault("use_proxy_for_cloud_metadata", false)
+	config.BindEnvAndSetDefault("remote_tagger_timeout", 30*time.Second)
```

Comment:
what's the unit that the user should expect? wondering if we should name it `remote_tagger_timeout_<unit>`

Reply:
It's a duration in Go's notation (so you should expect `30s`, `5m`, etc). I can rename it `remote_tagger_timeout_duration` if you think that's clearer.

Reply:
Looks like most duration config values in this file are integers, which imho less error prone.

We can do it this way `timeout := config.Datadog.GetInt("remote_tagger_timeout_seconds") * time.Second` To avoid any bad surprises. 

But no strong opinion tbh, we can ask for Agent Core's input.

Approving the PR :lgtm:

Reply:
@sgnn7 any thoughts on this?

Reply:
@ahmed-mez It's a tricky question to be honest and we had some cards come up about this. I think in this case I agree it should be parsed as ints (seconds) for consistency. No need to add the `_unit` since we don't do that for other similar fields. See AC-1303 for some discussion on this.